modules/govuk/manifests/deploy/config.pp

```diff
@@ -43,6 +43,11 @@
 # [*sidekiq_logfile*]
 #   Path to the logfile for Sidekiq (async work queue library).
 #   Default: 'log/sidekiq.log'
+#
+# [*govuk_sidekiq_json_logging*]
+#   Whether the app outputs logs in JSON format. Only applications running
+#   govuk_sidekiq > 6 will need this
+#   Default: false
```

Comment:
```suggestion
```

As per the previous comment that we don't actually have a variable for this and I don't think we need one.